### PR TITLE
Add some guide metadata for REST extensions

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

**Review by geoand — APPROVED** [review]

File: docs/src/main/asciidoc/qute-reference.adoc [hunk 1]
```diff
@@ -2537,27 +2537,28 @@ There are several ways to solve this problem:
 * Annotate the model class with `@io.quarkus.runtime.annotations.RegisterForReflection` to make the reflection-based value resolver work
 
 
-[[resteasy_integration]]
-=== RESTEasy Integration
+[[rest_integration]]
+=== [[resteasy_integration]] REST Integration
 
 If you want to use Qute in your Jakarta REST application, then depending on which Jakarta REST stack you are using, you'll need to register the proper extension first.
-If you are using the traditional `quakus-resteasy` extension, then in your `pom.xml` file, add:
```
Comment (holly-cummins) [inline docs/src/main/asciidoc/qute-reference.adoc:2544]: Ah, the traditional `quakus-resteasy` extension. :)

File: docs/src/main/asciidoc/qute-reference.adoc [hunk 2]
```diff
@@ -2537,27 +2537,28 @@ There are several ways to solve this problem:
...
 ----
 <dependency>
     <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-resteasy-qute</artifactId>
+    <artifactId>quarkus-rest-qute</artifactId>
 </dependency>
 ----
 
-If instead you are using Quarkus REST (formerly RESTEasy Reactive) via the `quarkus-rest` extension, then in your `pom.xml` file, add:
+If instead you are using the legacy RESTEasy Classic-based `quakus-resteasy` extension, then in your `pom.xml` file, add:
```
Comment (holly-cummins) [inline docs/src/main/asciidoc/qute-reference.adoc:2555]:
```suggestion
If instead you are using the legacy RESTEasy Classic-based `quarkus-resteasy` extension, then in your `pom.xml` file, add:
```

**Review by holly-cummins — CHANGES_REQUESTED** [review]
I made one flippant comment, and one serious (well, as serious as a trivial typo can be) suggestion.

File: docs/src/main/asciidoc/qute-reference.adoc (hunk 2 above)
Comment (gsmet) [reply docs/src/main/asciidoc/qute-reference.adoc:2555]: Funny thing is that I tried several times to type `quakus` to search for it and I had to type it three time before not having `quarkus` in the end.

**Pull request merged**